aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -430,8 +431,10 @@ fn execute_block_no_limit(
         BlockExecutorConfig {
             local: BlockExecutorLocalConfig {
                 concurrency_level,
-                allow_fallback: true,
+                allow_sequential_block_fallback: true,
                 discard_failed_blocks: false,
+                enable_block_stm_profiling: false,
+                block_stm_committer_backup: BlockSTMCommitterBackup::Default,
```

Comment:
This is e.g. so we can easily disable if needed, but should we also connect it to actual configs (similar to concurrency_level)? @igor-aptos @zekun000

Reply:
we have execution config here https://github.com/aptos-labs/aptos-core/blob/e292612324aa5a71d398d7c12748a78b6e50dcf1/config/src/config/execution_config.rs#L34

Reply:
plumbed the configs as per our discussion (debugger and executor benchmark have options, general config has flag for backup)